libp2p/pubsub/floodsub.py

```diff
@@ -10,6 +11,9 @@
 
 PROTOCOL_ID = TProtocol("/floodsub/1.0.0")
 
+logger = logging.getLogger("libp2p.pubsub.floodsub")
```

Comment:
Could you elaborate on the advantage to have `logger` as the global, instead of a `ClassVar` in `Floodsub`?

Reply:
Just copying the loggers added during interop (with logging level set to `DEBUG`) and I guess these are temporary. So we can update it all at once later (make it a classvar and unset logging level).